packages/domain/lib/src/models/people_model/crew_member.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:domain/src/models/people_model/person.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+import 'cast_and_crew_member.dart';
+
+part 'crew_member.g.dart';
+
+@JsonSerializable()
+class CrewMember extends CastAndCrewMember {
+  final String? job;
+  final List<String>? jobs;
+  final Person? person;
+
+  const CrewMember(
+    this.job,
+    this.jobs,
+    this.person,
+  ) : super(
```

Comment:
short super

Reply:
when I try to do short:
const CrewMember(
    super.job,
    super.jobs,
    super.person,
  );
then generation do not work (